### DISCUSSION TITLE
What is new user default password?

### DISCUSSION THREAD

**Discussion by mdr0414** [supabase/supabase#14428] [Questions] [answered]
Hi -- during my app development, I need to create new Supabase users (Authentication -> Add User -> Create New User -> Auto Confirm User (true)) ... but what is the pw set to when the new user is created?  tia _mark

**Comment by GaryAustin1** [comment, marked as answer]
You should have to enter a made up password there for the user.  Whatever you want.

**Reply by mdr0414** [reply]
Ha ... yeah just came back to try to remove this question.  Spaced out the auto-fill.  thx
